readme-en.md

```diff
@@ -132,8 +132,7 @@ Language: **[😎 English](readme-en.md)** • [한국어 (Korean)](readme-ko.md
 - [Study data for algorithm](http://d2.naver.com/news/4850628) - Korean.
 
 ## Competitions
-- [Google Code Jam](https://code.google.com/codejam) - Google’s largest coding competition, calls on programmers around the world to put their skills to the test by solving multiple rounds of algorithmic puzzles.
-- [Google Code Jam Kickstart](https://code.google.com/codejam/kickstart) - University students looking to develop their coding skills and potentially pursue a Google career.
```

Comment:
Should we use this one to replace kickstart?

https://codingcompetitions.withgoogle.com/kickstart

Reply:
Oh, thank you @safv19. I've looked around and didn't see this. Thanks for pointing me to this! I've fixed it!